ZombieSharp/TopDefender.cs

```diff
@@ -42,7 +42,21 @@ public void TopDenfederOnRoundEnd()
             if (ClientsDamage.Count <= 0)
                 return;
 
-            else if (ClientsDamage.Count < 3)
+            if (CVAR_EnableStats.Value)
+            {
+                foreach (var player in ClientsDamage)
+                {
+                    if (player.Key == null)
+                        continue;
+
+                    if (player.Key.IsBot || player.Key.IsHLTV)
+                        continue;
+
+                    StatsSetData(player.Key, player.Value.Damage, 0, player.Value.Infected).Wait();
+                }
+            }
+
+            if (ClientsDamage.Count < 3)
```

Comment:
what is 3? Is it some sort of magic number? Why `ClientsDamage.Count` should be less then 3? It is not clear

Reply:
Top damage and Top infector supposed to be only 3. This is for the case when they're less than 3 players in the server.

Reply:
Ok, then it's better to move this into some public static property so it will be more clear. Like
```cs
if (ClientsDamage.Count < TopDefenderConfig.MinRequiredPlayers)
```